src/main.rs

```diff
@@ -1066,7 +1066,7 @@ async fn make_app(current_dir: &std::ffi::OsString) -> Result<Command> {
                 .short('t')
                 .long("template")
                 .help("Template to create")
-                .value_parser(["blank", "chat", "echo", "fibonacci", "file-transfer"])
+                .value_parser(["blank", "chat", "echo", "fibonacci", "file-transfer", "hypr-echo"])
```

Comment:
Let's change name here and throughout to `hyperapp-echo`

We'll set a convention that all hyperapp templates are prefixed by `hyperapp-`